### PR TITLE
Update documentation with metadata search and tags

### PR DESCRIPTION
**Pull request by karlaspuldaro**
Addresses #1113 
 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: docs/source/user_guide/runtime-conf.md [hunk 1]
```diff
@@ -41,7 +41,8 @@ elyra-metadata install runtimes --schema_name=kfp \
        --cos_endpoint=http://minio-service.kubeflow:9000 \
        --cos_username=minio \
        --cos_password=minio123 \
-       --cos_bucket=test-bucket
+       --cos_bucket=test-bucket \
+       --tags=("my-kfp")
```
Comment (karlaspuldaro) [inline docs/source/user_guide/runtime-conf.md:45]: btw this command doesn't work 🙄
@kevin-bates is it possible to create a runtime with tags field from CLI?

File: docs/source/developer_guide/metadata.md [hunk 2]
```diff
@@ -72,7 +72,10 @@ And each metadata file looks like:
     "cos_endpoint": "http://weakish1.fyre.ibm.com:30427",
     "cos_username": "minio",
     "cos_password": "minio123",
-    "cos_bucket": "lresende"
+    "cos_bucket": "my_bucket",
```
Comment (lresende) [inline docs/source/developer_guide/metadata.md:75]: could we use test-bucket as in the other examples?

File: docs/source/user_guide/runtime-conf.md [hunk 3]
```diff
@@ -55,7 +56,8 @@ This produces the following content in `my_kfp.json`:
         "cos_endpoint": "http://minio-service.kubeflow:9000",
         "cos_username": "minio",
         "cos_password": "minio123",
-        "cos_bucket": "test-bucket"
+        "cos_bucket": "test-bucket",
+        "tags": ["my-kfp"]
```
Comment (lresende) [inline docs/source/user_guide/runtime-conf.md:60]: how about something more meaningful such as ["kfp","dev"] or ["kfp", "v1.1"]

**Review by lresende — APPROVED** [review]
LGTM

**Pull request merged**